packages/legend-studio/src/models/metamodels/pure/model/executionPlan/result/TDSColumn.ts

```diff
@@ -15,12 +15,12 @@
  */
 
 import type { DataType } from '../../packageableElements/domain/DataType';
-import type { PackageableElementReference } from '../../packageableElements/PackageableElementReference';
+import type { PackageableElementImplicitReference } from '../../packageableElements/PackageableElementReference';
```

Comment:
using implicit because the type was   PackageableElementImplicitReference and was not able to access the properties by using  PackageableElementReference @MauricioUyaguari